src/components/Movie.js

```diff
@@ -0,0 +1,14 @@
+import { NavLink } from "react-router-dom"
+import { Image } from "./Image"
+import { Description } from "./Description"
+import './Movie.css'
+
+export const Movie = (movie) =>{
+    console.log(movie.data.name)
+    return (
```

Comment:
Do we need Image and description as separate components? Use card to display one card of movie